common/src/IO/NodeWriter.cpp

```diff
@@ -75,7 +76,7 @@ namespace TrenchBroom {
         }
 
         void NodeWriter::writeMap() {
-            m_serializer->beginFile();
+            m_serializer->beginFile(std::vector<const Model::Node*>{&m_world});
```

Comment:
You don't need to specifiy the type here.

Reply:
Huh didn't know that (given that it's a worldnode -> node upcast)

Reply:
I had to double check it myself ;-)